src/core/client_channel/subchannel.cc

```diff
@@ -644,6 +713,20 @@ Subchannel::Subchannel(SubchannelKey key,
     channelz_node_->SetChannelArgs(args_);
     args_ = args_.SetObject<channelz::BaseNode>(channelz_node_);
   }
+  // Extract stats plugin group from channel args
+  stats_plugin_group_ =
```

Comment:
These fields can all be set in the initialization list above (line 681) rather than in the body of the ctor.